dist/common/scripts/scylla_util.py

```diff
@@ -361,7 +382,7 @@ def yum_uninstall(pkg):
 def apt_uninstall(pkg):
```

Comment:
why do we have function to uninstall things ?

Reply:
uninstall function is defined but not used from any scripts.
Maybe we can drop this.